docs/articles/commands/converters/built_in_converters.md

```diff
@@ -0,0 +1,59 @@
+---
+uid: articles.commands.converters.built_in_converters
+title: Built-In Converters
+---
+
+# Built-In Converters
+DSharpPlus provides a number of built-in argument converters for common types. An easy way to remember which converters should be available is ".NET Primitives + Discord Entities." This means that all primitive types in .NET are supported, as well as Discord entities such as `DiscordUser`, `DiscordChannel`, `DiscordRole`, and so on. For an up-to-date list of built-in converters, go to the source and view which files are available: [DSharpPlus.Commands/Converters](https://github.com/DSharpPlus/DSharpPlus/tree/master/DSharpPlus.Commands/Converters). Below is a list of some of the most common built-in converters, and their caveats/quirks:
+
+## .NET Primitives
+All .NET primitive types have built-in support. Each primitive type is parsed through `.TryParse`.
+
+### DateTime
+The `DateTime` converter does not exist and is not planned to be added. Instead, use `DateTimeOffset` for more accurate time handling.
+
+### Enum
+Each enum type that used in a command will have the generic version of `EnumConverter<T>` created at startup. The generic converter is as fast as 30ns per conversion, while the non-generic converter is 60ns per conversion. Enums with less than 25 members have built-in choice provider support. Enums with more than 25 members will have auto-complete support.
```

Comment:
careful with numbers without specifying exact hardware; and, why is this necessary to point out if all enums will use the generic converter anyway?

also, should be "### Enums"

Reply:
> and, why is this necessary to point out if all enums will use the generic converter anyway?

I feel the distinction is warranted since we have a generic and non-generic version of the converter. I don't think it's wrong to assume that the non-generic version is used by default. Reading that the generic version is used by default is a fun little piece of trivia that the user might like - I know I would.

Reply:
i would still implore you to get rid of the numbers and make it something like "which is 2x faster", if you insist on making any performance claim